usaspending_api/awards/models/financial_accounts_by_awards.py

```diff
@@ -17,6 +17,8 @@ class AbstractFinancialAccountsByAwards(DataSourceTrackedModel):
     parent_award_id = models.TextField(blank=True, null=True)
     fain = models.TextField(blank=True, null=True)
     uri = models.TextField(blank=True, null=True)
+    prior_year_adjustment = models.TextField(blank=True, null=True)
+    pa_reporting_key = models.TextField(blank=True, null=True)
```

Comment:
Since there was a question about what `pa_` stands for it might be good to make use of `help_text` here similar to this spot on Award model: https://github.com/fedspendingtransparency/usaspending-api/blob/master/usaspending_api/awards/models/award.py#L54